.vscode/settings.json

```diff
@@ -2,9 +2,9 @@
     "files.exclude": {
         "**/.git": true,
         "build*": true,
-        "doc/doxygen/html/": true,
-        "doc/doxygen/latex/": true,
-        "doc/doxygen/xml/": true,
-        "doc/sphinx/_build/": true
+        "doc/doxygen/html/": true
```

Comment:
@darioizzo seems like something else might have slipped in here?

Reply:
so also that is to be removed? 
        "doc/doxygen/html/": true

Reply:
No, what I mean is that the ``.vscode/settings.json`` file shouldn't show up at all in the PR diff because I don't think there are any modifications needed to it in this PR. What I complained about originally was a modification to the file which (I believe) is done automatically by VScode each time you open the workspace (because of some extension which is installed on your side I believe).

Maybe you can just copy paste the version from ``master``?